src/java/org/apache/cassandra/auth/IAuthorizer.java

```diff
@@ -80,12 +81,13 @@ void grant(AuthenticatedUser performer, Set<Permission> permissions, IResource r
      * @param permissions Set of permissions to revoke.
      * @param revokee Role from which to the permissions are to be revoked.
      * @param resource Resource on which to revoke the permissions.
+     * @return the permissions that have been successfully revoked.
```

Comment:
As before, we could mention that this is different to the `permissions` attribute, for example:
```suggestion
     * @return the permissions that have been successfully revoked, comprised by the requested permissions excluding
     * those permissions that were already not granted.
```